src/plugin/column-chooser/column.chooser.view.js

```diff
@@ -22,15 +22,37 @@ export class ColumnChooserView {
 			source: 'column.chooser',
 			execute: column => {
 				column.isVisible = !this.state(column);
+				column.children.forEach(child => child.isVisible = column.isVisible);
 			}
 		});
-
+		
+		this.toggleChild = new Command({
+			source: 'column.chooser',
+			execute: (parent, child) => {
+				const state = !this.state(child);
+				
+				child.isVisible = state;
+				
+				if (parent.children.some(c => this.state(c))) {
```

Comment:
Can you replace these two conditions by one?
if ```some``` will not find anything it returns false.